app.js

```diff
@@ -34,6 +34,7 @@ program
 	.option('-y, --sync [peers...]', 'peers list to sync with')
 	.option('-l, --log <level>', 'log level')
 	.option('-s, --snapshot <round>', 'verify snapshot')
+	.option('-t, --coverage', 'enable functional tests code coverage')
```

Comment:
Only functional tests?

If not: `enable functional tests code coverage` > `measure test code coverage`

If yes: `enable functional tests code coverage` > `measure functional test code coverage`